staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -307,8 +307,8 @@ func (u *Unstructured) GetOwnerReferences() []metav1.OwnerReference {
 
 func (u *Unstructured) SetOwnerReferences(references []metav1.OwnerReference) {
 	var newReferences = make([]map[string]interface{}, 0, len(references))
-	for i := 0; i < len(references); i++ {
-		newReferences = append(newReferences, setOwnerReference(references[i]))
+	for _, reference := range references {
+		newReferences = append(newReferences, setOwnerReference(reference))
```

Comment:
@kevin-wangzefeng @cblecker ok, this is value type. so cauth object copy? if this influence mary, i think keep it origin is better. and think you for review.😊 can i close this PR ?